airflow/www/views.py

```diff
@@ -1855,6 +1855,19 @@ def trigger(self, session=None):
             flash(f"Cannot create dagruns because the dag {dag_id} has import errors", "error")
             return redirect(origin)
 
+        recent_confs = (
+            session.query(DagRun.conf)
+            .filter(
+                DagRun.dag_id == dag_id,
+                DagRun.run_type == DagRunType.MANUAL,
+                DagRun.conf is not None,
+            )
+            .order_by(DagRun.execution_date.desc())
+            .limit(5)
+            .all()
+        )
+        recent_confs = [getattr(conf, "conf") for conf in recent_confs]
```

Comment:
Conf are not always json compatible. I believe they can be plain 'str'. check `def encode_dag_run` func. We may want to also handle this case. Maybe extract a reusable function that will handle the conf. (`json.dumps` if possible, otherwise str, and return `conf_is_json`, and skip the conf that are not json in the response)